olp-cpp-sdk-core/include/olp/core/network2/NetworkTypes.h

```diff
@@ -81,13 +85,17 @@ class NetworkStatus {
 
   /**
    * @brief Get request id.
+   * @return A valid RequestId in case request was successfull, \c
+   * RequestIdConstants::RequestIdInvalid otherwise.
    */
-  RequestId GetRequestId() const { return request_id_; }
+  const RequestId& GetRequestId() const { return request_id_; }
```

Comment:
Is there a need to have a const reference for the value of a primitive type?

Reply:
Not really, I just wanted to keep consistent with ApiResponse class.